.sprinkler.yaml

```diff
@@ -24,6 +24,9 @@ scheduler:
     # apiKeyName: "x-api-key"
     # apiKey: "changeme"
 
+cleanup:
+  scheduledWorkflowTimeout: "720h"
```

Comment:
Let's do it like the following, and move the scheduler unlock in cleanup too.

```
cleanup:
  # table names
  scheduledWorkflow: "720h"
  workflowSchedulerLock: "1h"
  workflowActivationLock: "1h"
```

Reply:
done.